contents/tutorials/android-analytics.md

```diff
@@ -0,0 +1,435 @@
+---
+title: "How to set up analytics in Android"
+date: 2024-02-29
+author: ["lior-neu-ner"]
+tags: ['feature flags']
+---
+
+import { ProductScreenshot } from 'components/ProductScreenshot'
+import EventsInPostHogLight from '../images/tutorials/android-analytics/events-light.png'
+import EventsInPostHogDark from '../images/tutorials/android-analytics/events-dark.png'
+import IdentifyLight from '../images/tutorials/android-analytics/identify-light.png'
+import IdentifyDark from '../images/tutorials/android-analytics/identify-dark.png'
+import InsightLight from '../images/tutorials/android-analytics/create-insights-light.png'
+import InsightsDark from '../images/tutorials/android-analytics/create-insights-dark.png'
+
+[Product analytics](/product-analytics) enable you to gather and analyze data about how users interact with your Android app. To show you how to set up analytics, in this tutorial we create a basic Android app, add PostHog, and use it to [capture events](/docs/product-analytics/capture-events) and [create insights](/docs/product-analytics/insights).
```

Comment:
Will mention Kotlin. Dont think we need to link to RN. People dont usually refer to RN apps as "android"

```suggestion
[Product analytics](/product-analytics) enable you to gather and analyze data about how users interact with your Android app. To show you how to set up analytics, in this tutorial we create a basic Android app with Kotlin, add PostHog, and use it to [capture events](/docs/product-analytics/capture-events) and [create insights](/docs/product-analytics/insights).
```